include/ck_tile/ops/gemm/pipeline/gemm_pipeline_ag_bg_cr_mem_custom_policy.hpp

```diff
@@ -0,0 +1,256 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2024, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include "ck_tile/core.hpp"
+#include "ck_tile/ops/gemm/block/block_gemm_as_bs_cr_default_policy.hpp"
+#include "ck_tile/ops/gemm/block/block_gemm_as_bs_cr.hpp"
+
+namespace ck_tile {
+
+// Default policy for GemmPipelineAGmemBGmemCRegV1
+
+// Maximum Global Memory throughput pipeline with >=32KB data in fly
```

Comment:
What is the difference of this policy compared to include/ck_tile/ops/gemm/pipeline/block_gemm_pipeline_agmem_bgmem_creg_v1_default_policy.hpp ? Can we customize mentioned block_gemm_pipeline_agmem_bgmem_creg_v1_default_policy  with listed parameters? Maybe we can avoid code duplication